Cargo.toml

```diff
@@ -172,9 +172,9 @@ wasm-metadata = "0.235"
 wasm-pkg-client = "0.11"
 wasm-pkg-common = "0.11"
 wasmparser = "0.235"
-wasmtime = "34.0.0"
-wasmtime-wasi = "34.0.0"
-wasmtime-wasi-http = "34.0.0"
+wasmtime = { git = "https://github.com/bytecodealliance/wasmtime", package = "wasmtime", branch = "release-35.0.0" }
```

Comment:
Is it correct to link to a `branch` (which is mutable) rather than a commit SHA?  This could result in different builds happening against different upstreams couldn't it?

Reply:
Good catch; resolving this as we'll update to use the crates.io release once it's out.